app/controllers/cities_controller.py

```diff
@@ -0,0 +1,23 @@
+from http import HTTPStatus
+
+from app.exceptions.state_exc import StateNotFoundError
+from app.services.cities_services import get_cities, get_city
+
+
+def cities():
+
+    cities = get_cities()
```

Comment:
The return inside `get_cities()` brings two items (the cities, and a status code), but you are fetching only for the first one, in line 11 you return just the cities, but since the status code is not explicitly it will return an OK.

In this case, you should get rid of the return in the function and implement in the `cities` function.